src/brs/props/PropertyServiceImpl.java

```diff
@@ -9,7 +9,7 @@
 public class PropertyServiceImpl implements PropertyService {
 
   private final Logger logger = LoggerFactory.getLogger(Burst.class);
-  private static final String LOG_UNDEF_NAME_DEFAULT = "{} undefined. Default: >{}<";
```

Comment:
This "undefined" message gets people confused. Several asked me if everything was ok because of this message, so a better alternative is given.